frontend/src/metabase/dashboard/containers/DashboardApp.jsx

```diff
@@ -74,7 +86,28 @@ const mapDispatchToProps = {
 
 @connect(mapStateToProps, mapDispatchToProps)
 @fitViewport
-@title(({ dashboard }) => dashboard && dashboard.name)
+@title(
+  ({
+    dashboard,
+    isLoadingDashCards,
+    totalDashCards,
+    cardsLoaded,
+    hasBeenSeen,
+    isLoadingDashCardsComplete,
+  }) => {
+    if (isLoadingDashCards) {
+      return {
+        title: `${cardsLoaded} / ${totalDashCards} loaded`,
```

Comment:
Please use a combination of `msgid` and `ngettext` functions from the `ttag` package to make this translatable. See an example [here](https://github.com/metabase/metabase/blob/ebf1dec4786177dfb5fb56f425ac122bae143e43/frontend/src/metabase/components/MetadataInfo/TableInfo/ColumnCount.tsx#L18)

Reply:
@kulyk I don't believe the string needs to be pluralized here, Could this also just be a `t` ?

Reply:
Oh, yeah, I guess so